jukebox/src/main/webapp/js/dbScript.js

```diff
@@ -0,0 +1,77 @@
+const NO_VIDEO = -1;
+
+var db = firebase.firestore();
+
+//Adds a song to the queue
+function addSongDB(song_ID, title, room_ID) {
```

Comment:
this could also be a part of the calling method - the only difference is the ROOM_ID, right?